packages/workflow/src/records/handler/download.ts

```diff
@@ -32,8 +32,11 @@ function getDownloadedOrAssignedExtension(
   status: TaskStatus
 ) {
   if (
-    inScope(authHeader, ['declare', 'recordsearch']) ||
-    (hasScope(authHeader, 'validate') &&
+    inScope(authHeader, [
+      SCOPES.RECORD_SUBMIT_INCOMPLETE,
+      SCOPES.RECORDSEARCH
+    ]) ||
```

Comment:
Let's do it this way instead:
if (you have recordsearch || (you don't have register & submitForApproval))